internal/app/dogma/cli_cmds.go

```diff
@@ -55,6 +55,11 @@ var streamingFlag = cli.BoolFlag{
 	Usage: "Specifies whether to keep watching the file",
 }
 
+var listenerFlag = cli.StringFlag{
+	Name:  "listener, l",
+	Usage: "Specifies the executable path that handles watch events",
```

Comment:
Would you mind adding an explanation that shows what kinds of environment variables are injected for the listener?

Reply:
I hesitated to do that here because this "Usage" field seems to show short text about the flag.
Instead, I added "Description" section to the watch command c9b577e

Reply:
It looks better. 👍